### ISSUE TITLE
[Bug] Fee estimates are very high

### ISSUE THREAD

**Issue by emlowe** [Chia-Network/chia-blockchain#14457]
### What happened?

Fee estimates are very high - hundreds of mojos per cost when the blocks are fairly empty.

```
'current_fee_rate': 454, 'estimates': [454, 454, 0], 'fee_rate_last_block': 7.621014857016043e-06, 'fees_last_block': 1000, 'full_node_synced': True, 'last_block_cost': 131216120, 'last_peak_timestamp': 1675212533, 'last_tx_block_height': 3186157, 'mempool_fees': 0, 'mempool_max_size': 550000000000, 'mempool_size': 0, 'node_time_utc': 1675212567, 'num_spends': 0, 'peak_height': 3186157, 'success': True, 'target_times': [60, 120, 300]}


  Mempool max cost: 550000000000 CLVM cost
      Mempool cost:            0 CLVM cost
     Mempool count:            0 spends
   Fees in Mempool:            0 mojos

Stats for last transaction block:
      Block height:      3186157
        Block fees:         1000 mojos
        Block cost:    131216120 CLVM cost
          Fee rate:    7.621e-06 mojos per CLVM cost

Fee Rate Estimates:
    1  minute: 454 mojo per CLVM cost
    2 minutes: 454 mojo per CLVM cost
    5 minutes: 0 mojo per CLVM cost
```

### Version

1.7.0 rc4

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
N/A
```

**Comment by arvidn** [comment]
I think we need a more objective metric for how well we predict the fees. I think we should run a simulation where we know exactly what transaction we submit to the mempool and at what fees, and then ensure the estimate is within some accuracy of the true necessary fee rate.

**Comment by emlowe** [comment]
Closing resolved in 1.7.0
